locations/spiders/opendata_mos_household_services_ru.py

```diff
@@ -98,39 +117,41 @@ def parse_row(self, row):
         item["lat"] = cells.get("Latitude_WGS84")
         item["lon"] = cells.get("Longitude_WGS84")
         item["extras"]["operator"] = cells.get("OperatingCompany")
-        self.parse_category(item, cells.get("TypeObject"))
         self.parse_phones(item, cells)
         self.parse_hours(item, cells)
-        # TODO: parse district
-        return item
+        return self.parse_category(item, cells.get("TypeObject"))
 
     def parse_category(self, item: Feature, category: str):
         if tags := CATEGORY_MAPPING.get(category):
             apply_category(tags, item)
+            return item
         else:
             self.crawler.stats.inc_value(f"atp/opendata_mos_ru/category/failed/{category}")
+            return None
 
     def parse_phones(self, item: Feature, cells: dict):
         if phones := cells.get("PublicPhone"):
             item_phones = []
             for phone in phones:
                 deleted = phone.get("is_deleted")
                 number = phone.get("PublicPhone")
-                if deleted == "0" and number != "нет телефона":
+                if deleted == 0 and number != "нет телефона":
                     item_phones.append(number)
             if item_phones:
                 item["phone"] = "; ".join(item_phones)
 
     def parse_hours(self, item: Feature, cells: dict):
+        # TODO: parse ClarificationOfWorkingHours
         if hours := cells.get("WorkingHours"):
             try:
                 oh = OpeningHours()
                 for hour in hours:
-                    if hour.get("is_deleted") == "0":
+                    if hour.get("is_deleted") == 0:
                         day = DAYS_RU.get(hour.get("DayOfWeek", "").title())
                         times = hour.get("Hours", "").split("-")
                         open, close = times[0], times[1]
                         oh.add_range(day, open, close)
                 item["opening_hours"] = oh.as_opening_hours()
             except Exception as e:
+                # TODO: 1,5k hours failed
```

Comment:
Do you have an example of the opening hours formats that failed? Perhaps `OpeningHours.add_ranges_from_string(..)` could help with parsing?

Reply:
Fixed.